### DISCUSSION TITLE
C-Programming

### DISCUSSION THREAD

**Discussion by The-Leo** [The-Leo/PROGRAMMING-NOTES#1] [Q&A] [answered]
Let's talk about data-types: What are the data-types in C programming?

**Comment by The-Leo** [comment]
I'll go first to set the pace. There are four main blocks of data types - Basic data types, Derived data types, Void data types and enumeration data types. Now we can go into the fine details.

**Comment by The-Leo** [comment]
Under basic data types, there is "int" which stores integer values. List some more

**Reply by streetprophet90** [reply, marked as answer]
int char float double

**Reply by streetprophet90** [reply]
pointers

**Comment by streetprophet90** [comment]
pointers
